config/default.json

```diff
@@ -2,7 +2,12 @@
   "appInsights": {
     "applicationId": "",
     "tenantId": "",
-    "endpoint": "https://api.applicationinsights.io/v1/apps"
+    "endpoint": "https://api.applicationinsights.io/v1/apps",
+    "subscriptionId": "",
```

Comment:
We don’t need these config change because appinsight's resource id is automatically detected, and azure data explorer is out of our scope.